src/script.js

```diff
@@ -1,19 +1,51 @@
 'use strict';
```

Comment:
Why the name of this file is `script`? It nothing to tell me what I can find here

Reply:
fixed